src/frontend/staff-portal/src/app/components/jj-workbench/jj-workbench-dashboard/jj-workbench-dashboard.component.ts

```diff
@@ -38,7 +42,12 @@ export class JjWorkbenchDashboardComponent implements OnInit {
     this.busyService.busy$.subscribe(i => this.busy = i);
     this.authService.userProfile$.subscribe(userProfile => {
       if (userProfile) {
-        this.jjAdminRole = this.authService.checkRole("admin-judicial-justice");
+        
+        // TCVP-1981 - only show tabs to users with permissions
+        this.hasAssignmentsPermission = this.authService.checkRoles(["admin-judicial-justice", "support-staff"]);
```

Comment:
This kind of defeats the permission based access control and kind of goes back to RBAC. Have we looked at the ability to fetch the user's permissions? I suspect we would have to enumerate on the server all know permissions and check access to return a list of permissions of a user.

Reply:
In some other apps I support (that dont use keycloak), we have an endpoint to fetch the user's permissions.

Reply:
This method just delegates to the keycloak.isUserInRole(string) method. We could instead use the this.keycloak.getUserRoles() and manually enumerate over the results ourselves in javascript. Is this what you were thinking?

![image](https://github.com/bcgov/jag-traffic-courts-online/assets/55215368/dc1c821b-5485-446d-b0fe-4a3b2dcbdeaf)

Reply:
The goal here is to identify which roles the user has and using the permission matrix on the wiki identify which tabs the user is permitted to see. 
Open to an alternative implementation.

Reply:
> This kind of defeats the permission based access control and kind of goes back to RBAC.

I'm not sure I follow. I thought this is doing exactly permission based access control. Each tab in the UI is protected against the list of roles that permit that tab from displaying.